assets/templates/partials/feedback.tmpl

```diff
@@ -6,10 +6,10 @@
                 Yes
             </a>
             <span>|</span>
-            <a id="feedback-form-no" class="js-toggle" href="/feedback?service=cmd" aria-label="No I didn't find this page useful">
+            <a id="feedback-form-no" class="js-toggle" href="/feedback" aria-label="No I didn't find this page useful">
                 No
```

Comment:
This file should also be using dictionary tags for the text values and the aria label text.